pages/common/comby.md

```diff
@@ -0,0 +1,20 @@
+# comby
+
+> A tool for structural code search and replace that supports ~every language.
+> More information: <https://github.com/comby-tools/comby>.
+
+- Match and rewrite templates, and display any changes on the console:
```

Comment:
We don't use `console`, we could use `stdout`
```suggestion
- Match and rewrite templates, and print changes:
```
which template?

Reply:
Actually I copied from `tldr rustfmt`: Check a file for formatting and display any changes on the console

> which template?

It's `comby -in-place {{match_pattern}} {{rewrite_pattern}}`, so `'assert_eq!(:[a], :[b])'` is a template. I want to show how this template look like so used an example instead... Especially for the case with `:[b].Capitalize`